ros2_api:
  ros__parameters:
    joints_topic: "/output" # Where to read this topic. Optional. Default: ... (Message is always all five arrays)
    command_interfaces: # MAYBE EXTEND WITH END EFFECTOR POS AND TRAJECTORY 
      interfaces: # Which command interfaces are allowed to be received
        - "velocity"
        - "position"
        - "effort"
      velocity_topic: "/velocity_cmd" # Where to send the velocity command to. Optional. Only allowed if in interfaces list. Default: ...
      position_topic: "/position_cmd" # Where to send the position command to. Optional. Only allowed if in interfaces list. Default: ...
      effort_topic: "/effort_cmd" # Where to send the effort command to. Optional. Only allowed if in interfaces list. Default: ...
			
    transport:
      type: "ros2_api/UnixDomainSockets" # Name of the plugin as set in the pluginlib.xml
      params: # Your own params for the plugin specified. Not checked by parser. Passed to the plugin. Optional.
        your: "own"
        yaml: "structure"